### PR TITLE
Update Cargo.toml file on all contracts.

### PR DESCRIPTION
**Pull request by fnando**
### What

Simplify all Cargo.toml files, matching what's generated by `soroban contract init`.

### Why

`soroban contract init --with-example <name>` uses this repo as the source. By making these changes, we are able to copy examples that are closer to the actual generated contract.

### Known limitations

N/A

### CODE REVIEW

File: account/Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 [package]
 name = "soroban-account-contract"
 version = "0.0.0"
-authors = ["Stellar Development Foundation <info@stellar.org>"]
-license = "Apache-2.0"
```
Comment (leighmcculloch) [inline account/Cargo.toml:5]: The examples are authored by SDF and licensed as Apache 2 so I think we should probably keep them specified as such? Or if we drop the author at least keep the license. I guess we have the license in the root so it doesn't really matter what's in the toml. 

Lgtm 👍

**Review by leighmcculloch — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@ target/
 /.sccache/
 .DS_Store
 .soroban
+errors/test_snapshots/
```
Comment (kalepail) [inline .gitignore:5]: Should we ignore all test snapshot directories?

File: account/Cargo.toml (hunk 1 above)
Comment (kalepail) [reply account/Cargo.toml:5]: I vote toss. If we're going to use these as templates we should simplify the tomls to help downstream folks not accidentally be adding authors and licenses they don't intend to

**Review by kalepail — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (leighmcculloch) [reply .gitignore:5]: Yup with `**/test_snapshots/`.

**Pull request merged**